src/hotspot/share/jvmci/jvmci_globals.hpp

```diff
@@ -157,6 +157,11 @@ class fileStream;
           "error data to this file"                                         \
           "[default: ./" LIBJVMCI_ERR_FILE "] (%p replaced with pid)")      \
                                                                             \
+  product(bool, LibJVMCICompilerThreadHidden, true, EXPERIMENTAL,           \
```

Comment:
This sounds more like a diagnostic flag than experimental.

Reply:
It kind of is but because of the way the way [JVMCIGlobals::enable_jvmci_product_mode ](https://github.com/tkrodriguez/jdk/blob/tkr-hidden-fix/src/hotspot/share/jvmci/jvmci_globals.cpp#L169) works it seems easier to just make it product like all the other special JVMCI flags.